doc/news/2020-05-26_0.9.2.md

```diff
@@ -50,7 +50,7 @@ Get started with Elektra by running `docker run -it elektra/elektra`.
 We removed the `maintained` status of the following [plugins](https://www.libelektra.org/plugins/readme):
 
 - blockresolver
-- csvstorage
+- CSV storage
```

Comment:
I think this change should be reverted, since it refers to a specific plugin name.
https://www.libelektra.org/plugins/csvstorage